src/main/java/carpet/utils/MobAI.java

```diff
@@ -93,4 +98,24 @@ public static TrackingType byName(String aspect)
         }
     }
 
+    public static void jump(LivingEntity e){//Most *REDACTED* up code in the world
```

Comment:
oh, boi - use vanilla code, just get access to that via an interface...

Reply:
wdym by use and interface...

Reply:
one example: https://github.com/gnembon/fabric-carpet/blob/94ffd5b9900653f1a2994bf051046790a87e5168/src/main/java/carpet/helpers/XPcombine.java#L40

Reply:
idk how to do

Reply:
ok, will merge as is, and I will fix the accessors